controller/upload_file.rst

```diff
@@ -243,6 +243,16 @@ logic to a separate service::
 
             return $fileName;
         }
+
+        public function getTargetDir()
+        {
+            return $this->targetDir;
+        }
+
+        public function setTargetDir($targetDir = '')
```

Comment:
I would just remove the setter. IMO there's no need to have it. We only need the getter to be able to read the configured directory later.